### PR TITLE
Tests with Postgres

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,29 +12,41 @@ on:
...
 
                 include:
                     -
                         php: 7.4
                         symfony: ""
                         currency-bundle: "^1.7"
+                    -
+                        php: 8.0
+                        symfony: "^5.3"
+                        postgres: "13.3"
```
Comment (lchrusciel) [inline .github/workflows/build.yml:31]: I'm wondering if we shouldn't test v14 as well. Also, if we are testing with postgres, shouldn't we trigger build with MySQL? At the end, we are testing SqlLite only right now

**Review by lchrusciel — COMMENTED** [review]
I'm a little bis sceptical about the current stage of this PR and if it is needed. Anyway, it requires rebase

**Comment by lchrusciel** [comment]
Ok, I've was just convinced about need of this PR with #184

File: .github/workflows/build.yml (hunk 1 above)
Comment (loic425) [reply .github/workflows/build.yml:31]: Yes I can add the v14 for Postgres, but I think the MySQL build can be added in another PR.

Comment (loic425) [reply .github/workflows/build.yml:31]: done for PostgreSQL v14

**Comment by lchrusciel** [comment]
Thanks Loic! 🎉

**Pull request merged**